eng/Build.props

```diff
@@ -1,6 +1,12 @@
 <Project>
   <Import Project="$(MSBuildThisFileDirectory)Common.props" />
   <ItemGroup Condition="'$(CreateArchives)' == 'true'">
+    <ProjectToBuild Include="$(RepoRoot)src\archives\pkgs\AzureBlobStorage\AzureBlobStorage-archive.proj">
```

Comment:
Long term: sharing common binaries across rids/architectures

Reply:
As recently discussed, sharing the application binaries between multiple platform publishes is not really a supported scenario for .NET projects. I understand the intent here to minimize the payload of the extension.

One way to mitigate this is to be able to produce an extension as platform neutral and have `dotnet monitor` launch it using `dotnet`.